app/popup.js

```diff
@@ -19,22 +20,31 @@ function updateValue(e) {
 
 // When user click on "Add"
 function setWebsite() {
+  let error = false;
   chrome.storage.local.get(function (result) {
     if (
       typeof result['websites'] !== 'undefined' &&
       result['websites'] instanceof Array
     ) {
-      result['websites'].push(log);
-      for (const r in result['websites']) {
-        myURLsRedirect.push('*://*.' + result['websites'][r] + '/*');
+      // Check if website is already in list
+      if (result['websites'].includes(log) === true) {
```

Comment:
Here, you don't necessarily need to check if it's `=== true` as `Array.includes` returns a Boolean.

Reply:
Done in new commit